src/base/bittorrent/session.cpp

```diff
@@ -323,6 +325,16 @@ namespace
         return {};
     }
 #endif
+
+    using ListType = lt::entry::list_type;
+
+    ListType setToEntryList(const QSet<QString> &input)
```

Comment:
I would move this function above `convertIfaceNameToGuid()` such that functions that are conditional are placed at bottom.